File: backend/MediaVerse/MediaVerse.Domain/Entities/Episode.cs
namespace MediaVerse.Domain.Entities;

public partial class Episode
{
    public Guid Id { get; set; }

    public Guid SeriesId { get; set; }

    public string Synopsis { get; set; } = null!;

    public int SeasonNumber { get; set; }

    public int EpisodeNumber { get; set; }

    public virtual Entry IdNavigation { get; set; } = null!;

    public virtual Series Series { get; set; } = null!;
}

File: frontend/src/models/entry/episode/Episode.ts
import { Entry } from '../Entry.ts'
import { WorkOn } from '../WorkOn.ts'
import { EntryPreview } from '../../author/Author.ts'

export interface Episode {
  entry: Entry,
  synopsis: string,
  seasonNumber: number,
  episodeNumber: number,
  workOns: WorkOn[],
  series: EntryPreview
}
